react/style-guide.md

```diff
@@ -157,8 +157,8 @@ src
 ## Naming
 
   - **Extensions**: Use `.js` extension for React components.
-  - **Filename**: Use PascalCase for filenames. E.g., `ReservationCard.js`.
-  - **Reference Naming**: Use PascalCase for React components and camelCase for their instances. eslint: [`react/jsx-pascal-case`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-pascal-case.md)
+  - **Filename**: For component filenames and services use PascalCase. E.g., `ReservationCard.js`.
```

Comment:
What would be the reason why we use pascal case for services and not camel case?

Reply:
I feel like Services are full objects themselves (even if they aren't actually objects). I prefer doing `import UserService from '~services/users'` and then doing `UserService.foo()` than doing `import userService from '~services/users'` and then doing `userService.foo()`. CamelCase feels like a simple variable/function while PascalCase feels like a full entity. 

Of course, it's a matter of preference I guess. I call a vote on this one.